run.sh

```diff
@@ -32,8 +32,9 @@ java $JAVA_OPTS \
  -Drestolino.classes=$RESTOLINO_CLASSES \
  -Drestolino.packageprefix=$PACKAGE_PREFIX \
  -DSTART_EMBEDDED_SERVER=N \
-  -javaagent:zebedee-cms/target/dependency/aws-opentelemetry-agent-1.32.0.jar \
-  -Dotel.propagators=tracecontext,baggage \
+ -javaagent:zebedee-cms/target/dependency/aws-opentelemetry-agent-1.31.0.jar \
```

Comment:
Why is this downgrading the otel agent? I thought you put in the upgrade to sort some issues?

Reply:
Thanks for catching this, I made the update now and tested Zebedee with Babbage locally which shows the content being displayed without any errors!